scripts/release/audit_rosdistro.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+
+# Copyright 2014-2020 Open Source Robotics Foundation, Inc.
```

Comment:
Nitpick: was code for this new module copied from somewhere> If yes, from where? If no, the year should probably just be 2020.